barretenberg/ts/src/main.ts

```diff
@@ -71,6 +71,32 @@ async function init(bytecodePath: string, crsPath: string, subgroupSizeOverride
   return { api, acirComposer, circuitSize, subgroupSize };
 }
 
+async function initUltraHonk(bytecodePath: string, crsPath: string, subgroupSizeOverride = -1) {
```

Comment:
could do away with this new init function by just using the existing one. The only difference is that this one doesn't create an acirComposer